DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/README.md

```diff
@@ -0,0 +1,318 @@
+
+## License
+This code sample is licensed under MIT license.
+
+## Building the `shannonization` Tutorial
+### Include Files
+The included header `dpc_common.hpp` is located at `%ONEAPI_ROOT%\dev-utilities\latest\include` on your development system.
+
+### Running Samples in DevCloud
+If running a sample in the Intel DevCloud, remember that you must specify the compute node (fpga_compile or fpga_runtime) as well whether to run in batch or interactive mode. For more information see the Intel® oneAPI Base Toolkit Get Started Guide ([https://devcloud.intel.com/oneapi/get-started/base-toolkit/](https://devcloud.intel.com/oneapi/get-started/base-toolkit/)).
+
+When compiling for FPGA hardware, it is recommended to increase the job timeout to 12h.
+
+### On a Linux* System
+ 
+1. Generate the `Makefile` by running `cmake`.
+     ```
+   mkdir build
+   cd build
+   ```
+   To compile for the Intel® PAC with Intel Arria® 10 GX FPGA, run `cmake` using the command:  
+    ```
+    cmake ..
+   ```
+   Alternatively, to compile for the Intel® PAC with Intel Stratix® 10 SX FPGA, run `cmake` using the command:
+ 
+   ```
+   cmake .. -DFPGA_BOARD=intel_s10sx_pac:pac_s10
+   ```
+ 
+2. Compile the design through the generated `Makefile`. The following build targets are provided, matching the recommended development flow:
+ 
+   * Compile for emulation (fast compile time, targets emulated FPGA device): 
+     ```
+     make fpga_emu
+     ```
+   * Generate the optimization report: 
+     ```
+     make report
+     ```
+   * Compile for FPGA hardware (longer compile time, targets FPGA device): 
+     ```
+     make fpga
+     ```
+3. (Optional) As the above hardware compile may take several hours to complete, an Intel® PAC with Intel Arria® 10 GX FPGA precompiled binary can be downloaded <a href="https://software.intel.com/content/dam/develop/external/us/en/documents/shannonization.fpga.tar.gz" download>here</a>.
+ 
+### On a Windows* System
+Note: `cmake` is not yet supported on Windows. A build.ninja file is provided instead. 
```

Comment:
Ah, I see why there is some Ninja code.
Curious this CMake configuration is not working yet...